### ISSUE TITLE
Conversations - In Person

### ISSUE THREAD

**Issue by pqhf5kd** [monicahq/monica#2027]
Would it be possible to add 'In Real Life' to the 'How did you communicate?' dropdown on the 'Record a new conversation' page?

For recording important conversations that doesn't deserve/fit the format of an Activity entire.

**Comment by scheidm** [comment]
If you go to Settings > Personalization and add 'Real Life' to contact type, you can then add it for a conversation. However it will record the conversation as 'messages', so there's no way to put in an overview. I made a bug for that over here: https://github.com/monicahq/monica/issues/2005

**Comment by djaiss** [comment]
Yes @scheidm this is kind of a workaround but it's not "great". We need to add a checkbox at the top saying `This was done in person` or something.

**Comment by dmurvihill** [comment]
+1 I would like this.

**Comment by emansom** [comment]
Would appreciate this feature as well.

My usecase would be for it to log all in-persons conversations I have with people; recorded, transcribed and tagged by a companion Android app automatically.

I think this is achieveable with modern ML models like Whisper and other ML based transcribers.

Due to my autism, I often have trouble recalling conversations in detail if they weren't related to my special interests. Monica could aid here.

Thinking aloud here, not meant as "devloper pls fix"; more meant as collaborative brainstorming.
